bundled-libs/XML/RPC.php

```diff
@@ -1433,7 +1433,9 @@ function parseResponse($data = '')
 
         $encoding = $this->getEncoding($data);
         $parser_resource = xml_parser_create($encoding);
-        $parser = 1;
```

Comment:
This was a scalar value, maybe copy pasta mistake?

Reply:
That was a mistake I did in https://github.com/s9y/Serendipity/commit/a6619f77cd840f2c1888524052aa3c07c0b645b4#r152820624. I needed a replacement for `(int) $parser_resource;`, and though this could just be hardcoded to a number. That might even be feasible, but not when changing it only at one of the places that creates `$parser` like this. The replacement is the point of this PR, to use `$parser = is_resource($parser_resource) ? ((int) $parser_resource) : spl_object_hash($parser_resource);` everywhere.